src/main/java/com/rocketden/main/dto/game/GameDto.java

```diff
@@ -0,0 +1,15 @@
+package com.rocketden.main.dto.game;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+import com.rocketden.main.dto.room.RoomDto;
+
+@Getter
+@Setter
+public class GameDto extends RoomDto {
```

Comment:
I'm thinking that this will want to have the same fields as `RoomDto`. I initially just but `RoomDto` as a field within `GameDto`, but then realized that seemed unnecessary if I could just extend it. Thoughts?